ejb-timer/.github/quickstart_ejb_timer_ci.yml

```diff
@@ -0,0 +1,21 @@
+name: WildFly EJB Timer Quickstart CI
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths:
+      - <quickstart-path>/**'
+      - .github/workflows/quickstart_ci.yml
+
+jobs:
+  call-quickstart_ci:
+    uses: ./.github/workflows/quickstart_ci.yml
+    with:
+      QUICKSTART_PATH: ejb-timer
+      DEPLOYMENT_DIR: webapp
+      TEST_PROVISIONED_SERVER: true
+      TEST_BOOTABLE_JAR: true
```

Comment:
this quickstart does not uses bootable jar, thus this should be removed (default value is false)

Reply:
fixed